.github/workflows/ci.yml

```diff
@@ -97,3 +97,51 @@ jobs:
 
       - name: Run Command
         run: composer ci:php:${{ matrix.command }}
+
+  unit-tests:
+    name: Unit tests
+
+    runs-on: ubuntu-latest
+
+    needs: [php-lint, composer-validate]
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.0
+          - 7.1
+          - 7.2
+          - 7.3
+          - 7.4
+        dependencies:
+          - lowest
+          - highest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: ${{ matrix.php-version }}
+          ini-values: xdebug.max_nesting_level=500
```

Comment:
Do we need this (we did not previously change the setting)?

Reply:
Yes. Without this setting, we'd get a "maximum nesting level reached" error in `CssInlinerTest::inlineCssAppliesMoreSpecificCssSelectorToMatchingElements`:
https://github.com/MyIntervals/emogrifier/runs/539483776?check_suite_focus=true